templates/iot.html

```diff
@@ -1,332 +1,416 @@
+                    </div>
+                    <div class="col-6 col-medium-3">
+                        <p>
+                            Snapcraftは、ソフトウェアをSnapとしてコンテナ化するためのコマンドラインツールです。Snapは、組み込みアプリケーションのデプロイを目的としてコンテナ化されたパッケージであり、他のテクノロジーと比べてパフォーマンスに優れています。Ubuntu
+                            Coreのアーキテクチャはsnapをベースとし、その利益を最大限に生かしています。
+                        </p>
+                        <hr class="is-muted" />
+                        <p>
+                            <a class="p-button--positive" href="/contact-us?product=iot">お問い合わせ</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
         </div>
-        <div class="col-4 p-card">
-          <h3 class="p-card__title">安全性と信頼性</h3>
-          <P class="p-card__content">トランザクション下でのアップデートによってフェイルセーフアップグレードが保証され、機器の安全性が確保されます。定期的な信頼性の高いアップデートにより健全なエコシステムと規制に対するコンプライアンスが確保されます。</P>
+        <div class="row">
+            <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+                <hr class="is-muted" />
+                <div class="row">
+                    <div class="col-3 col-medium-2">
+                        <h3 class=p-heading--5>Snap Store</h3>
+                    </div>
+                    <div class="col-6 col-medium-3">
+                        <p>
+                            CanonicalのSnap
+                            Storeにより、開発者によるデバイスのソフトウェア更新を容易にし、これにより信頼性の高いOTA更新、デルタ更新など、確実な更新を実現します。特定の顧客向けのプライベートマーケットプレイスを作成することで、自社デバイスの高度なソフトウェア管理にも対応できます。
+                        </p>
+                        <hr class="is-muted" />
+                        <p>
+                            <a class="p-button--positive" href="/contact-us?product=iot">お問い合わせ</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
         </div>
-        <div class="col-4 p-card">
-          <h3 class="p-card__title">多様なハードウェア</h3>
-          <P class="p-card__content">ARMからIntelまで、お客様の次期デバイスに適した基板があります。プロトタイプを短時間で作成し、すぐに標準リファレンスモジュールと標準Linuxプラットフォームで開発が可能です</P>
+        <div class="row">
+            <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+                <hr class="is-muted" />
+                <div class="row">
+                    <div class="col-6  col-start-large-4 col-medium-3 col-start-medium-3">
+                        <p>デバイス向けUbuntu Pro</p>
+                        <p>
+                            プリインストールに適したワンタイムライセンスでご提供。指定のLTSバージョンに対応する25,000以上のOSSに対するセキュリティメンテナンスを最長10年間ご提供。
+                            Ubuntuをデバイスへプリインストールして再配布することが可能となります。
+                        </p>
+                        <p>対象：IoT/組み込みデバイスのみとなります。</p>
+                        <hr class="is-muted" />
+                        <p>
+                            <a class="p-button--positive"
+                               href="https://docs.google.com/document/d/1kKk7sRq21je9sJC93-6JjTqhNbR-Eh3WRBCJqH44i00/edit#heading=h.g8y1nsq6bjmw">詳細情報</a>
```

Comment:
This should link to the "Ubuntu Pro for devices" section below, current link just leads to the doc